### ISSUE TITLE
Reusable block only selected with arrow keys after "icon click"

### ISSUE THREAD

**Issue by johanhermansson** [WordPress/gutenberg#29332]
So I usually use arrow keys to select blocks and parent blocks, and when I want to remove a reusable block it needs to be easy to select it, with mouse click or arrow keys.

https://user-images.githubusercontent.com/1567642/109124737-902c3180-774b-11eb-923e-cf4936f32ef5.mov

###  How to reproduce
* Create reusable block
* Use the block on a page
* Reload editor page in browser
* Try to select reusable block by stepping with arrow keys
* Press block icon to select reusable block
* Use arrow keys to step between block
* Now reusable block should be selectable

**Comment by annezazu** [comment]
@johanhermansson Thanks for reporting this! Can you confirm what versions of WordPress and Gutenberg you are using?

**Comment by johanhermansson** [comment]
@annezazu No problem :) Version 10.0.2

**Comment by johanhermansson** [comment]
I already had problems with this on 9.9.x, I think, but it worked on 9.8.4 before the new parent block highlight functionality

**Comment by Mamaduka** [comment]
Hi, @johanhermansson

I'm no longer able to reproduce this issue on Gutenberg `trunk`. Unfortunately, I couldn't track down the PR that fixed the issue, but it's safe to assume that it was fixed in the latest versions or will be in the next one.

https://user-images.githubusercontent.com/240569/121128397-626f8200-c83c-11eb-89f3-c4a0c11e1567.mp4

**Comment by johanhermansson** [comment]
@Mamaduka Yes, it works in Gutenberg plugin 10.7.4, but I still have the problem in Wordpress 5.7.2 when only using core block editor. Great that it's on it's way to next core version. Thanks! 👍 

So what's the workflow here? Should I close this issue? 🙃

**Comment by Mamaduka** [comment]
> So what's the workflow here? Should I close this issue? 🙃

I will close this issue as resolved. Feel free to re-open it or create a new one if you stop anything similar in the future.
